src/analyzer/tests/referrer-policy.js

```diff
@@ -72,11 +75,13 @@ export function referrerPolicyTest(
 
   // Find the last known valid policy value in the referrer policy
   let policy = output.data
-    .split(",")
-    .filter((e) => valid.includes(e.toLowerCase().trim()))
-    .reverse()[0]
-    ?.toLowerCase()
-    .trim();
+    ? output.data
+        .split(",")
+        .filter((e) => valid.includes(e.toLowerCase().trim()))
+        .reverse()[0]
+        ?.toLowerCase()
+        .trim()
+    : "";
```

Comment:
Might it be a bit neater to do?
`let policy = output.data?.split(",").etc(...) ?? "";`